sleap_io/io/main.py

```diff
@@ -47,7 +47,7 @@ def save_slp(
     return slp.write_labels(filename, labels, embed=embed)
 
 
-def load_nwb(filename: str) -> Labels:
+def load_nwb(filename: str, format: str) -> Labels:
```

Comment:
Suggestion: since we only have two "formats", it might be easier to just make this a boolean. For example:

```py
def load_nwb(filename: str, as_training: bool | None = None):
    if as_training is None:
        as_training = ...  # try to auto-detect based on whether there are tracks or user-labeled instances

    if as_training:
        return nwb.read_nwb_training(filename)
    else:
        return nwb.read_nwb(filename)
```

And maybe we want to rename `nwb.read_nwb` to `nwb.read_nwb_predictions` for clarity?

Reply:
PS: We probably want to keep `nwb.read_nwb` and anything else that existed before as an alias for backwards compatibility.